x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/cards/malware_protections_card.test.tsx

```diff
@@ -60,48 +65,69 @@ describe('Policy Malware Protections Card', () => {
     );
   });
 
-  it('should set Blocklist to disabled if malware is turned off', () => {
-    const expectedUpdatedPolicy = cloneDeep(formProps.policy);
-    setMalwareMode(expectedUpdatedPolicy, true);
-    render();
-    userEvent.click(renderResult.getByTestId(testSubj.enableDisableSwitch));
-
-    expect(formProps.onChange).toHaveBeenCalledWith({
-      isValid: true,
-      updatedPolicy: expectedUpdatedPolicy,
+  const features: Array<{ name: 'blocklist' | 'onWriteScan'; config: string; default: boolean }> = [
+    { name: 'blocklist', config: 'blocklist', default: true },
+    { name: 'onWriteScan', config: 'on_write_scan', default: true },
+  ];
+  for (const feature of features) {
```

Comment:
`it.each()`? :)

Reply:
good point, I always forgot it, because `jest` supports it while `mocha` (and thus `cypress`) does not, and I mixed up which framework I am using at the moment ヽ(。_°)ノ

d6cf40a6e6fbd9f223df1b7929fe7627984fe10f

Reply:
thank you for the thorough testing! 🙌